doctr/datasets/vocabs.py

```diff
@@ -15,6 +15,11 @@
     'punctuation': string.punctuation,
     'currency': '£€¥¢฿',
     'ancient_greek': 'αβγδεζηθικλμνξοπρστυφχψωΑΒΓΔΕΖΗΘΙΚΛΜΝΞΟΠΡΣΤΥΦΧΨΩ',
+    'arabic_letters': 'ءآأؤإئابةتثجحخدذرزسشصضطظعغـ‌‍‎‏فقكلمنهوىي',
+    'arabic_farsi_letters': 'ٯٰپچژڜڢڤڥڧڨکگی',
+    'hindi_numbers': '٠١٢٣٤٥٦٧٨٩',
```

Comment:
Could you rename it `hindi_digits` please? :pray:

Reply:
Hi @fg-mindee, I believed I have made all requested changes. Can you please double check and let me know if there is anything I missed?

Reply:
Thanks! I'll run the CI checks to ensure we don't miss anything!